src/index.ts

```diff
@@ -557,6 +559,31 @@ Please refer to the Monika documentations on how to how to configure notificatio
 
     throw error
   }
+
+  async deprecationHandler(config: Config) {
+    let showMessage = false // if probes.alerts using `query` convert to `assertion`
+    if (config && config.probes) {
+      config.probes.forEach(function (probe: Probe, iprobe: number) {
+        if (probe.alerts) {
+          probe.alerts.forEach(function (alert: ProbeAlert, ialert: number) {
+            if (alert.query !== undefined && alert.assertion === undefined) {
+              config.probes[iprobe].alerts[ialert] = {
+                assertion: alert.query,
+                message: alert.message,
+              }
+              showMessage = true
+            }
+          })
+        }
+      })
+    }
+
+    if (showMessage) {
+      log.warn(
+        'Monika config "alerts.query" was Deprecated! please change into "alerts.assertion"'
```

Comment:
"is deprecated. Please use `alerts.assertion`"

Reply:
warning message update